### ISSUE TITLE
Some bugs with demo

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#181]
```
I'm just running demo provided by that project and found some issues.

Before
1. Launch demo: java -jar gwtupload-samples-0.6.6.war 
2.go to page "Single uploader"

Issue #1
1. By pressing image button (3rd option) select file with wrong mime type. ie 
text file.
2. Select correct file type

What is the expected output? What do you see instead?
Progress bar isn't showed.

Issue #2
1. Refresh page in browser
2. In 2nd option select file with correct mime type. 
3. Press "Send" button and after progress bar is showed press stop icon (cross 
icon)
4. Press "Send" button.

What is the expected output? What do you see instead?
Upload isn't started.


What version of the product are you using? On what operating system?
Env - FF 22


Please provide any additional information below.


```

Original issue reported on code.google.com by `SVMaslyu...@gmail.com` on 6 Aug 2013 at 10:27

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Could you test these issues in this instance? :
http://gwtupload.alcala.org
```

Original comment by `manuel.carrasco.m` on 25 Oct 2013 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 8cb2db5a5df4.
```

Original comment by `manuel.carrasco.m` on 31 Dec 2013 at 4:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 31 Dec 2013 at 4:34
- Added labels: **Milestone-1.0.0**
